DQM/CTPPS/python/ctppsPixelDQMSource_cfi.py

```diff
@@ -0,0 +1,9 @@
+import FWCore.ParameterSet.Config as cms
+
+ctppsPixelDQMSource = cms.EDAnalyzer("CTPPSPixelDQMSource",
+#    tagRPixDigi = cms.InputTag("RPixDetDigitizer", ""),
+    tagRPixDigi = cms.InputTag("ctppsPixelDigis", ""),
+    tagRPixCluster = cms.InputTag("clusterProd", ""),  
+    RPStatusWord = cms.untracked.uint32(0x8000), # rpots included in readout
+    verbosity = cms.untracked.uint32(1)
```

Comment:
Hi @popovvp - this needs to be changed to 0 for production use. Thanks for a quick PR fixing it.